build/actions/scan.js

```diff
@@ -23,14 +23,19 @@ dockerVersionProperties = ['Version', 'ApiVersion'];
 module.exports = {
   signature: 'scan',
   description: 'Scan for resinOS devices in your local network',
-  help: '\nExamples:\n\n	$ rdt scan\n	$ rdt scan --verbose',
+  help: '\nExamples:\n\n	$ rdt scan\n	$ rdt scan --timeout 120\n	$ rdt scan --verbose',
```

Comment:
I think it will make our lives easier (and also reduce unnecessary noise during code reviews) if we generate JS only once before publishing (e.g. check [`0.0.6`](https://github.com/resin-os/resin-device-toolbox/commit/f8f2eddc394af53e0b8ab1e6203dc8b76dc05dba) and [`0.0.5`](https://github.com/resin-os/resin-device-toolbox/commit/3f7b982d54292c7f8decc23f7fcc3935b125c37d)) . We already do this on the backend services and the supervisor as well (generating JS once in production, that is)